webroot/src/components/Lists/Pagination/Pagination.spec.ts

```diff
@@ -42,315 +46,115 @@ describe('Pagination component', async () => {
-    //     expect(pages[3]).to.equal(4);
-    // });
-    // it('should have the same pages when pagination ids are same between different components', async () => {
-    //     const spy = sinon.spy();
-    //     const wrapper1 = await mountShallow(Pagination, {
-    //         props: {
-    //             paginationId: 'test2',
-    //             pageChange: spy,
-    //             listSize: 38,
-    //         }
-    //     });
-    //     const wrapper2 = await mountShallow(Pagination, {
-    //         props: {
-    //             paginationId: 'test2',
-    //             pageChange: spy,
-    //             listSize: 38,
-    //         }
-    //     });
-    //     const instanceOne = wrapper1.vm;
-    //     const instanceTwo = wrapper2.vm;
-    //
-    //     expect(instanceOne.currentPage).to.equal(instanceTwo.currentPage);
-    //     expect(instanceOne.pageSize).to.equal(instanceTwo.pageSize);
-    //     expect(spy.calledTwice).to.be.true;
-    //
-    //     spy.resetHistory();
-    //     instanceOne.setPage(3);
-    //
-    //     expect(instanceOne.currentPage).to.equal(instanceTwo.currentPage);
-    //     expect(instanceOne.pageSize).to.equal(instanceTwo.pageSize);
-    //     expect(spy.calledOnce).to.be.true;
-    //
-    //     spy.resetHistory();
-    //     instanceOne.setSize({ value: 20 });
-    //
-    //     expect(instanceOne.currentPage).to.equal(instanceTwo.currentPage);
-    //     expect(instanceOne.pageSize).to.equal(instanceTwo.pageSize);
-    //     expect(spy.calledOnce).to.be.true;
-    // });
-    // it('should set the page to the last page when the page size change creates a larger page than exists', async () => {
-    //     const spy = sinon.spy();
-    //     const wrapper = await mountShallow(Pagination, {
-    //         props: {
-    //             paginationId: 'test3',
-    //             pageChange: spy,
-    //             listSize: 38,
-    //         }
-    //     });
-    //     const instance = wrapper.vm;
-    //
-    //     instance.setPage(4);
-    //     instance.setSize({ value: 50 });
-    //
-    //     const thirdSpy = spy.thirdCall;
-    //
-    //     expect(thirdSpy.calledWith(0, 50)).to.be.true;
-    //     expect(instance.currentPage).to.equal(1);
-    // });
+    it('should load with previous next paging', async () => {
+        const spy = sinon.spy();
+        const wrapper = await mountFull(Pagination, {
+            props: {
+                paginationId: 'test',
+                pageChange: spy,
+                pagingPrevKey: 'test-prev',
+                pagingNextKey: 'test-next',
+            },
+        });
+        const instance = wrapper.vm;
+        const pages = instance.pages.map((page) => page.displayValue);
+
+        expect(pages).to.be.an('array');
+        expect(pages).to.have.length(1);
+        expect(pages[0]).to.equal(1);
+        expect(wrapper.findComponent(LeftCaretIcon).exists(), 'previous arrow').to.equal(false);
```

Comment:
My assumption is that, since `pagingPrevKey` is non-null, there is a previous page. In that case, I would expect this left/previous arrow to exist. Where am I off in my mental model?

Reply:
We're not showing the previous arrow when we're on page 1.

Reply:
Right. If we're on page one, though, would there ever be a non-null "previous key", though?

Reply:
I think we were just trying to sneak in a little non-happy-path testing for things that are de-coupled (store & components). I updated it so it's just the happy path and should read a little easier.

Reply:
Got it, totally fair! That makes sense.